controls/views.py

```diff
@@ -122,6 +122,271 @@ def controls_selected(request, system_id):
+
+            # Responsible Entities
+            c = ws.cell(row=1, column=10, value="Responsible Entities")
+            c.border = Border(right=Side(border_style="thin", color="444444"), bottom=Side(border_style="thin", color="444444"), outline=Side(border_style="thin", color="444444"))
+
+            # Control Owner(s)
+            c = ws.cell(row=1, column=11, value="Control Owner(s)")
+            c.border = Border(right=Side(border_style="thin", color="444444"), bottom=Side(border_style="thin", color="444444"), outline=Side(border_style="thin", color="444444"))
+
+            # Type ["System-Specific", "Hybrid", "Inherited", "Common", "blank"]
+            c = ws.cell(row=1, column=12, value="Type")
+            c.border = Border(right=Side(border_style="thin", color="444444"), bottom=Side(border_style="thin", color="444444"), outline=Side(border_style="thin", color="444444"))
+
+            # Inherited From
+            c = ws.cell(row=1, column=13, value="Inherited From")
+            c.border = Border(right=Side(border_style="thin", color="444444"), bottom=Side(border_style="thin", color="444444"), outline=Side(border_style="thin", color="444444"))
+
+            # Provide As ["Do Not Share", "blank"]
+            c = ws.cell(row=1, column=14, value="Provide As")
+            c.border = Border(right=Side(border_style="thin", color="444444"), bottom=Side(border_style="thin", color="444444"), outline=Side(border_style="thin", color="444444"))
+
+            # Evaluation Status ["Evaluated", "Expired", "Not Evaluated", "Unknown", "blank"]
+            c = ws.cell(row=1, column=15, value="Evaluation Status")
+            c.border = Border(right=Side(border_style="thin", color="444444"), bottom=Side(border_style="thin", color="444444"), outline=Side(border_style="thin", color="444444"))
+
+            # Control Origination
+            c = ws.cell(row=1, column=16, value="Control Origination")
+            c.border = Border(right=Side(border_style="thin", color="444444"), bottom=Side(border_style="thin", color="444444"), outline=Side(border_style="thin", color="444444"))
+
+            # History
+            c = ws.cell(row=1, column=17, value="History")
+            c.border = Border(right=Side(border_style="thin", color="444444"), bottom=Side(border_style="thin", color="444444"), outline=Side(border_style="thin", color="444444"))
+
+        wb.save('export.xlsx')
+
+        with NamedTemporaryFile() as tmp:
+            wb.save(tmp.name)
+            tmp.seek(0)
+            stream = tmp.read()
+            blob = stream
+
+        mime_type = "application/octet-stream"
+        filename = "{}_control_implementations.xlsx".format(system.root_element.name.replace(" ","_"))
+        
+        resp = HttpResponse(blob, mime_type)
+        resp['Content-Disposition'] = 'inline; filename=' + filename
+        return resp
+
```

Comment:
Lines 378-385 are after the `return` statement at line 377.  Is that what you want?